get-involved.php

```diff
@@ -99,11 +99,11 @@
         <div class="half">
             <p>elementary OS is created and used by people from all around the World; help us make the experience even better by translating it into more languages.</p>
             <p>elementary OS uses an open, collaborative translation tool on Launchpad called Rosetta. <a href="/docs/translation-guide#translating-applications" class="read-more">Learn More</a></p>
```

Comment:
Yeah we never used Transifex for apps either. Transifex only was used for the website